src/components/pages/system/HomePage.jsx

```diff
@@ -19,6 +19,7 @@ const HomePage = () => {
   const [apiUsageData, setApiUsageData] = useState({});
   const [topKvedData, setTopKvedData] = useState([]);
   const [topCompanyTypeData, setTopCompanyTypeData] = useState([]);
+  const history = useHistory();
```

Comment:
HomePage is render by react-router, so history located in props
![image](https://user-images.githubusercontent.com/38906004/102657876-23ab1a00-417f-11eb-8507-d1a9aac9b5ea.png)